### PR TITLE
chore(authors): Adding a gulp task to generate an AUTHORS file

### LINKED ISSUE #27: Make authors file

### PR DESCRIPTION
**Pull request by pdehaan**
Fixes #27

### CODE REVIEW

**Comment by 6a68** [comment]
@pdehaan mind configuring `committers` to also include the email address?

File: gulpfile.js [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
+  data = require('gulp-data'),
+  imagemin = require('gulp-imagemin'),
+  jade = require('gulp-jade'),
+  jshint = require('gulp-jshint'),
+  livereload = require('gulp-livereload'),
+  minifycss = require('gulp-minify-css'),
+  notify = require('gulp-notify'),
+  rename = require('gulp-rename'),
+  sass = require('gulp-sass'),
+  uglify = require('gulp-uglify');
```
Comment (johngruen) [inline gulpfile.js]: way to alphabetize, bro.

Comment (pdehaan) [inline gulpfile.js]: I have a sickness.

**Pull request merged**